assets/src/scss/pages/_page.scss

```diff
@@ -89,12 +89,71 @@ div.page-template {
   }
 }
 
-.error-message {
-  color: $orange-hover;
-}
+.password-form {
+  .input-text{
+    font-size:16px;
+    line-height:1.6;
+    font-weight:400;
+    margin-bottom:16px;
+    position:relative;
+  }
+
+  .input-text input{
+    border:1.5px solid $grey-20;
+    border-radius:4px;
+    display:inline-block;
+    font-size:16px;
+    padding:15px 14px 8px;
+    min-width:400px;
 
-.password-field {
-  padding-top: $space-xs;
-  padding-bottom: $space-xs;
-  margin-bottom: $space-xs;
+    &.is-invalid {
+      border-color: $dark-orange;
+      background: $white;
+
+      &:focus {
+        border-color: $dark-orange;
+      }
+    }
+  }
+
+  .input-text input:hover{
+    border:1px solid $grey-40;
+  }
+
+  .input-text input:focus{
+    border-color:rgba(0,109,253,0.75);
+    outline:0 none;
+  }
+
+  .invalid-pw-feedback {
```

Comment:
I think the font family for the error message should be Roboto, at the moment it's Lora

Reply:
Yep, that's correct, thanks @mleray